### PR TITLE
metrics: Add sysbench performance test

### LINKED ISSUE #7485: metrics: add sysbench performance test

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds the sysbench performance test for kata CI.

Fixes #7485

### CODE REVIEW

**Review by cmaf — APPROVED** [review]
thanks @GabyCT

File: tests/metrics/density/k8s-sysbench.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	check_cmds "${cmds[@]}"
+
+	# Check no processes are left behind
+	check_processes
+
+	export pod_name="test-sysbench"
+
+	kubectl create -f "${SCRIPT_PATH}/runtimeclass_workloads/sysbench-pod.yaml"
+	kubectl wait --for=condition=Ready --timeout=120s pod "$pod_name"
+
```
Comment (dborquez) [inline tests/metrics/density/k8s-sysbench.sh:55]: Please remove empty line

**Review by dborquez — CHANGES_REQUESTED** [review]

File: tests/metrics/density/k8s-sysbench.sh (hunk 1 above)
Comment (GabyCT) [reply tests/metrics/density/k8s-sysbench.sh]: @dborquez changes applied thanks

**Review by dborquez — APPROVED** [review]
LGTM, thank you @GabyCT

**Pull request merged**